### PR TITLE
Update external-dns to v0.6.0

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

File: charts/rancher-external-dns/v0.0.3/README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# external-dns
+
+## Chart Details
+
+This chart will do the following:
+
+* Create a deployment of [external-dns] within your Kubernetes Cluster.
+
+Currently this uses the [Zalando] hosted container, if this is a concern follow the steps in the [external-dns] documentation to compile the binary and make a container. Where the chart pulls the image from is fully configurable.
+
```
Comment (prachidamle) [inline charts/rancher-external-dns/v0.0.3/README.md:10]: @StrongMonkey maybe we should update the ReadMe to mention the source of the image is the official release from https://github.com/kubernetes-sigs/external-dns

**Review by prachidamle — CHANGES_REQUESTED** [review]

File: charts/rancher-external-dns/v0.0.3/README.md (hunk 1 above)
Comment (StrongMonkey) [reply charts/rancher-external-dns/v0.0.3/README.md:10]: :+1:

Comment (StrongMonkey) [reply charts/rancher-external-dns/v0.0.3/README.md:10]: @prachidamle Updated.

**Review by prachidamle — APPROVED** [review]
LGTM

**Comment by deniseschannon** [comment]
@StrongMonkey Can you link this to the issue?

**Pull request merged**